gsoc/background.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title:  "Background on GSoC: start here!"
+show_main: false 
+---
+
+# Background: Start Here!
+
+### What is GSoC? 
+
+Google Summer of Code is a remote summer coding internship program funded by 
```

Comment:
The definition on the [GSOC floss manual](http://write.flossmanuals.net/gsocstudentguide/what-is-google-summer-of-code/) doesn't say internship. It's also something NumFOCUS has been discussing how to call this. I would write it as in the manual:
```
... is a global program that matches students up with open source,...
```

Reply:
In fact... [the explicitly said it's not an inthernship](https://developers.google.com/open-source/gsoc/faq).